app/views/profile/show.html.haml

```diff
@@ -0,0 +1,9 @@
+.inset
+  %h2= "Welcome, #{current_user.full_name} <#{current_user.email_address}>"
+  = form_tag(me_path(current_user), method: :post) do
+    %label.string.control-label First Name
+    %input.form-control{type: 'text', name: 'first_name', value: current_user.first_name}
```

Comment:
I guess this is an aesthetic choice, not a technical one.

I chose `/me` because it was short, pithy and unambiguous. It represents the `current_user` publicly to the user, regardless of how that current user is represented privately to the application. Like `/proposals` returns a list of proposals related to the `current_user`, `/me` returns a list of attributes that describe the `current_user`.

It feels somehow more personal than simply a resource to be managed.

The URL is the same for all users, like Google does with https://aboutme.google.com/

If it would be more clear internally, we could rename the controller to `MeController` or rename the route to `/profile`, if that's helpful.